user/api_jwtValidator_test.go

```diff
@@ -0,0 +1,133 @@
+package user
```

Comment:
it was only because I was was testing the Validator via the `api` but yeah I might just do that